### ISSUE TITLE
Add button/tab to open CovJSON by URL

### ISSUE THREAD

**Issue by tomkralidis** [covjson/playground#3]
As part of the OGC API - Coverages (OACov) work, some implementations (such as https://github.com/geopython/pygeoapi) are adding support for CoverageJSON output.  It would be great to add support to the playground for a user to copy/paste a URL whose output is a CoverageJSON document.

**Comment by letmaik** [comment]
You mean this?

https://covjson.org/playground/#https://raw.githubusercontent.com/covjson/playground/master/public/coverages/grid-tiled.covjson

It's an easter egg...

**Comment by tomkralidis** [comment]
Excellent! https://covjson.org/playground/#https://dev.api.weather.gc.ca/coverages-demo/collections/gdps-temperature/coverage?subset=Long(-67,-52)&subset=Lat(46,60)&rangeSubset=1

The Easter egg is a great capability, thanks!  It would be useful to have a "from URL" tab type thing for user interaction if possible.

**Comment by letmaik** [comment]
I agree, it's a useful feature that's unnecessarily hidden. Your URL doesn't work because the server doesn't send the right CORS headers. Are you fixing that?

**Comment by tomkralidis** [comment]
Try now?

**Comment by letmaik** [comment]
Nope, but I'm confused why you're asking me to try. Does it already work for you? I tried on Firefox and Chrome. The error is "Multiple CORS header ‘Access-Control-Allow-Origin’ not allowed".

**Comment by tomkralidis** [comment]
ok, works for me fwiw (fixed double header response)

**Comment by letmaik** [comment]
Opening by URL through UI is implemented now and will be part of the next playground release:
![image](https://user-images.githubusercontent.com/530988/148401030-16ec4092-873a-4be4-92ef-9dd916220996.png)
